Makefile

```
@@ -21,7 +21,9 @@ export GOFLAGS?=-mod=readonly -trimpath
 BUILD_DATE=$(shell if hash gdate 2>/dev/null; then gdate --rfc-3339=seconds | sed 's/ /T/'; else date --rfc-3339=seconds | sed 's/ /T/'; fi)
 GITCOMMIT=$(shell git log -1 --pretty=format:"%H")
 GITTAG=$(shell git describe --tags --always)
+DEFAULT_STABLE=$(shell curl -SsL https://dl.k8s.io/release/stable.txt)
```

Let's use a versioned version marker because this might point to a version that we do not support.

```suggestion
DEFAULT_STABLE=$(shell curl -SsL https://dl.k8s.io/release/stable-1.24.txt)
```

---

done